examples/blog/config.js

```diff
@@ -248,7 +248,7 @@
                 nga.field('id').label('ID'),
                 nga.field('name'),
                 nga.field('published', 'boolean').cssClasses(function(entry) { // add custom CSS classes to inputs and columns
-                    if (entry.values.published) {
+                    if (!!entry.values.published) {
```

Comment:
Make it `if(entry.values && !!entry.values.published)` to avoid a warning when first loading the page